install_config/install/advanced_install.adoc

```diff
@@ -315,6 +315,34 @@ node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 ====
 
+Nodes should have a label signifying the router. The registry should be
+installed on them. For example:
```

Comment:
This part sounds confusing to me.  Nodes have labels that often indicate their region.  A convention is to set the 'region=infra' label for the router and registry selector.